RELEASE.md

```diff
@@ -0,0 +1,17 @@
+# Publishing a New Release
+*This page contains instructions for admins of this project to release a new version.*
+
+## Run Makefile
+Run `$ make publish VERSION_NUMBER` at the root directory of needle (where the `Makefile` is located)
+
+For example:
+```
+$ make publish 0.13.0
+```
+
+This runs the steps specified in the `Makefile`.
+
+The `Makefile` does not support failure recovery yet. If a certain step fails, you might want to figure out which step failed, and manually run the remaining steps.
```

Comment:
Add 'a' before failure. 
'you might want to' sounds casual, maybe better state 'it needs to be resolved and the remaining steps should be run manually.'